docs/content/sftp.md

```diff
@@ -279,6 +279,10 @@ option before first run, the auto-detection will be skipped, and if
 you set a different value later this will override any existing.
 Value `none` can be set to avoid any attempts at executing shell
 commands, e.g. if this is not allowed on the server.
+However, if you have “shell_type = none” in the configuration then you
+should not have an ssh command such as
+`ssh = ssh -i path\ssh_id user@ip-address`, you should have `ssh = `
```

Comment:
To any possible confusion, where one is reading this as they *must* explicitly set the `ssh` option empty, maybe just say it must not be set? Maybe even link to its documentation `[ssh](#sftp-ssh)` - which also have a relevant note on the same topic:

> Rclone ignores all the internal config if you use this option and expects you to configure the ssh binary with the user/host/port and any other options